src/utils/naturalCompare.ts

```diff
@@ -0,0 +1,22 @@
+export const naturalCompare = (a: any, b: any) => {
```

Comment:
What's this about?

Reply:
That ensures that the sort returns a natural sort of the alphanumerical strings. For example: 
```
[ 123asd, 19asd, 1234asd ]
``` 
returns
```
[ 123asd, 1234asd, 19asd ]
```
instead of 
```
[ 1234asd, 123asd, 19asd ]
```